GTMAppAuthSwift/Sources/KeychainAttribute.swift

```diff
@@ -0,0 +1,365 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Foundation
+import Security
+// Ensure that we import the correct dependency for both SPM and CocoaPods since
+// the latter doesn't define separate Clang modules for subspecs
+#if SWIFT_PACKAGE
+import AppAuthCore
+import GTMSessionFetcherCore
+#else
+import AppAuth
+import GTMSessionFetcher
+#endif
+
+/// A helper providing a concrete implementation for saving and loading auth data to the keychain.
+@objc(GTMKeychainStore)
+public final class KeychainStore: NSObject {
```

Comment:
Shouldn't the file be named KeychainStore?

Reply:
You're right! Also caught in the swift-api rebase.